alerts/vault/vault-token-renew-revoke-alert.v1.json

```diff
@@ -8,19 +8,22 @@
   "conditions": [
     {
       "displayName": "New condition",
-      "conditionMonitoringQueryLanguage": {
+      "conditionPrometheusQueryLanguage": {
         "duration": "0s",
-        "query": "fetch gce_instance\n| { metric 'workload.googleapis.com/vault.token.renew.time'\n; metric 'workload.googleapis.com/vault.token.revoke.time' }\n| outer_join 0\n| group_by 1m\n| condition val(0) > 1000 || val(1) > 1000",
-        "trigger": {
-          "count": 1
-        }
+        "evaluationInterval": "30s",
+        "query": "rate({\"workload.googleapis.com/vault.token.renew.time\", monitored_resource=\"gce_instance\"}[1m]) > 1000\nor\nrate({\"workload.googleapis.com/vault.token.renew.time\", monitored_resource=\"gce_instance\"}[1m]) > 1000"
```

Comment:
these both say renew.time; one should be revoke.time

Reply:
Also it looks like these are both Gauges, so rate() is an invalid operator: https://cloud.google.com/monitoring/api/metrics_opsagent#opsagent-vault

avg_over_time is more appropriate